lib/spree_springboard/resource/export/order.rb

```diff
@@ -34,10 +34,19 @@ def after_sync(order)
         def springboard_invoice!(order)
           if springboard_can_invoice?(order)
             invoice_springboard_id = springboard_invoice_create!(order)
-
             springboard_invoice_line_items_create!(order, invoice_springboard_id) if invoice_springboard_id.present?
             springboard_invoice_complete!(order) if invoice_springboard_id.present?
             invoice_springboard_id
+          # If the order has been invoiced AND it's open:
+          # attempt to resync line_items and complete the order.
+          elsif springboard_invoiced(order)
```

Comment:
Is this duplicated from the other PR?

Reply:
Yes, it has been resolved after rebasing off master and merging: https://github.com/JenniKayne/spree_springboard/pull/8/commits/cd396a13427b46a2578ee0cef50db12f50160878